orca-clouddriver/src/test/kotlin/com/netflix/spinnaker/orca/clouddriver/pipeline/KubernetesPreconfiguredJobSpec.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.clouddriver.pipeline
+
+import com.netflix.spinnaker.orca.api.test.OrcaFixture
+import com.netflix.spinnaker.orca.api.test.orcaFixture
+import com.netflix.spinnaker.orca.clouddriver.KatoRestService
+import com.netflix.spinnaker.orca.clouddriver.model.TaskId
+import com.ninjasquad.springmockk.MockkBean
+import dev.minutest.junit.JUnit5Minutests
+import dev.minutest.rootContext
+import io.mockk.every
+import io.mockk.verify
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.http.MediaType
+import org.springframework.test.context.TestPropertySource
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers
+
+class KubernetesPreconfiguredJobSpec : JUnit5Minutests {
+
+  fun tests() = rootContext<Fixture> {
+    orcaFixture {
+      Fixture()
+    }
+
+    context("kubernetes preconfigured job") {
+      test("the application has a preconfigured job") {
```

Comment:
I might not use the word "application" since people might think you're talking about a Spinnaker application.

Maybe `"preconfigured jobs endpoint returns job definition"`.

Reply:
done